src/azul/__init__.py

```diff
@@ -792,14 +793,32 @@ def aggregation_lambda_timeout(self, *, retry: bool) -> int:
 
     service_lambda_timeout = 15 * 60
 
-    api_gateway_timeout = 29
+    _api_gateway_timeout = 29
 
     # The number of seconds to extend the timeout of a Lambda fronted by API Gateway so that API Gateway times out
     # before the Lambda. We pad the Lambda timeout so we get consistent behaviour. Without this padding we'd have a
     # race between the Lambda being killed and API Gateway timing out.
     #
     api_gateway_timeout_padding = 2
```

Comment:
We should also inline this attribute, if possible, and move the comment to the body of api_gateway_lambda_timeout.

Reply:
Not possible. There are other usages. `api_gateway_timeout` could be inlined however.
![image](https://user-images.githubusercontent.com/48401926/159344596-47555396-51f0-4665-b855-304b1a909186.png)